CONTRIBUTING.md

```diff
@@ -1,5 +1,109 @@
 # Contributing to Zitadel
 
+Zitadel is an open-source identity and access management platform built with a modern tech stack including Go (API), Next.js/React (login), Angular (Console), and Docusaurus (docs) - all orchestrated through an Nx monorepo with pnpm for efficient development workflows.
```

Comment:
Can we please settle with a naming?

IMO we should use Login, Api, Docs, Console

Reply:
done